.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+### Description
```

Comment:
i don't like it actually, since i think the desc is somekind of useless.

Reply:
+1. We discussed this earlier and decided to omit pull request templates.